packages/infobox/components/InfoBox.tsx

```diff
@@ -67,10 +67,14 @@ export class InfoBox extends React.PureComponent<InfoBoxProps, {}> {
       onDismiss,
       className
     } = this.props;
-
     const hasActions = primaryAction || secondaryAction;
-
     const isUrgentMessage = appearance === "danger" || appearance === "warning";
+    const dataCy = [
+      "infoBox",
+      ...(appearance && appearance !== "default"
```

Comment:
saw this pattern a couple of times now and realized that i would have lazily gone with having `appearance.default` and justifying it to myself with the slightly lighter footprint on implementation cost (yes, the conditional). 

no action item, just wanted to share